lib/features/sign_up/view_model/sign_up_bloc.dart

```diff
@@ -57,5 +57,8 @@ class SignUpBloc extends Bloc<SignUpEvent, SignUpState> {
         emit(AuthenticationFailed());
       }
     });
+    on<InitialiseState>((event, emit) {
```

Comment:
This state is used set the state back to the initial state. 
The problem is: when the user cancel sign in multiple time, the snack bar which is now defined in line 34 in the sign_up_view file, are not able to listen to the state again therefore the snack bar can not be displayed since the second attempt